dotcom-rendering/src/components/TopBar.importable.tsx

```diff
@@ -0,0 +1,194 @@
+/**
+ * @file
+ * This file is rework of https://github.com/guardian/dotcom-rendering/blob/94957ccdefe180ce6c292cb81de528cb616676e5/dotcom-rendering/src/components/HeaderTopBar.importable.tsx
+ * and includes the previously separated out `HeaderTopBarPrintSubscriptions` and `HeaderTopBarSearchJobs` components.
+ * This is intended to replace the existing `HeaderTopBar` component after being AB tested.
+ */
+import { css } from '@emotion/react';
+import { from, space } from '@guardian/source-foundations';
+import { Hide } from '@guardian/source-react-components';
+import { useEffect, useState } from 'react';
+import { pageSkinContainer } from '../layouts/lib/pageSkin';
+import { addTrackingCodesToUrl } from '../lib/acquisitions';
+import { center } from '../lib/center';
+import type { EditionId } from '../lib/edition';
+import { nestedOphanComponents } from '../lib/ophan-helpers';
+import { useAuthStatus } from '../lib/useAuthStatus';
+import { usePageViewId } from '../lib/usePageViewId';
+import { palette as themePalette } from '../palette';
+import { useConfig } from './ConfigContext';
+import { TopBarLink } from './TopBarLink';
+import { TopBarMyAccount } from './TopBarMyAccount';
+
+interface TopBarProps {
+	editionId: EditionId;
+	idUrl?: string;
+	mmaUrl?: string;
+	discussionApiUrl: string;
+	idApiUrl: string;
+	hasPageSkin?: boolean;
+}
+
+const topBarStyles = css`
+	background-color: ${themePalette('--masthead-top-bar-background')};
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+
+	height: 52px;
+	box-sizing: border-box;
+	padding: 0 10px;
+
+	${from.mobileLandscape} {
+		padding: 0 ${space[5]}px;
+	}
+
+	${from.tablet} {
+		height: 60px;
+	}
+
+	${from.desktop} {
+		height: 64px;
+		justify-content: flex-end;
+	}
+`;
+
+const topBarStylesWithoutPageSkin = css`
+	${from.wide} {
+		padding-right: 96px;
+	}
+`;
+
+const topBarLinkContainerStyles = css`
+	height: 100%;
+
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+`;
+
+const paddingRightStyles = css`
+	padding-right: ${space[3]}px;
+`;
+
+const verticalDividerStyles = css`
+	${from.desktop} {
+		:before {
+			content: '';
+			border-left: 1px solid
+				${themePalette('--masthead-top-bar-vertical-divider')};
+			display: flex;
+			position: relative;
+			height: 38px;
+		}
+	}
+`;
+
+const VerticalDivider = () => <div css={verticalDividerStyles} />;
+
+const TopBarLinkContainer = ({
+	isLastChild = false,
+	children,
+}: {
+	isLastChild?: boolean;
+	children: React.ReactNode;
+}) => (
+	<div css={[topBarLinkContainerStyles, !isLastChild && paddingRightStyles]}>
```

Comment:
A very small not to say that [according to the Emotion best practice, if you want to vary a single CSS property it's best to use the `styles` prop](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).